libra-node/src/main_node.rs

```diff
@@ -67,10 +69,10 @@ fn setup_debug_interface(config: &NodeConfig) -> Runtime {
         "{}:{}",
         config.debug_interface.address, config.debug_interface.admission_control_node_debug_port,
     )
-    .to_socket_addrs()
-    .unwrap()
-    .next()
-    .unwrap();
+        .to_socket_addrs()
```

Comment:
haha, I'm gonna send out an easy PR to fix this nonsense, we should be extracting a socket address directly from the config...